services/movie.py

```diff
@@ -23,18 +29,19 @@ def get_movie_by_id(movie_id: int) -> Movie:
 
 
 def create_movie(
-    movie_title: str,
-    movie_description: str,
-    genres_ids: list = None,
-    actors_ids: list = None,
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
 ) -> Movie:
-    movie = Movie.objects.create(
-        title=movie_title,
-        description=movie_description,
-    )
-    if genres_ids:
-        movie.genres.set(genres_ids)
-    if actors_ids:
-        movie.actors.set(actors_ids)
-
-    return movie
+    with transaction.atomic():
```

Comment:
Please, apply `transaction.atomic()` as a function decorator instead of a context manager since all function body should be treated as an atomic transaction